src/components/Connect.vue

```diff
@@ -93,17 +93,37 @@ const handleDisconnect = () => {
 }
 
 const addEventListeners = () => {
-  window.ethereum?.on('accountsChanged', handleAccountsChanged)
-  window.ethereum?.on('chainChanged', handleChainChanged)
-  window.ethereum?.on('connect', handleConnect)
-  window.ethereum?.on('disconnect', handleDisconnect)
+  ;(window.ethereum as any)
+    ?.on(
+      'accountsChanged',
+      handleAccountsChanged
+    )(window.ethereum as any)
+    ?.on(
+      'chainChanged',
+      handleChainChanged
+    )(window.ethereum as any)
+    ?.on(
+      'connect',
+      handleConnect
+    )(window.ethereum as any)
+    ?.on('disconnect', handleDisconnect)
```

Comment:
This formatting is the result of `yarn lint:format:fix` but looks quite strange.

Reply:
indeed, i wrote it a bit differently